ansible/environments/jenkins/group_vars/openwhisk-vm1-he-de

```diff
@@ -36,3 +36,4 @@ runtimes_enable_concurrency: true
 limit_action_concurrency_max: 500
 
 invoker1_machine: openwhisk-vm3-he-de
+invoker_use_runc: false
```

Comment:
@houshengbo do you have plans to update Jenkins VMs to a higher Docker version once this PR is merged? This PR will update Docker to a higher version on the CLI side so that we needed to switch off runc usage in the invoker...